recipes/jill/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "jill" %}
+{% set version = "0.11.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jill-{{ version }}.tar.gz
+    sha256: 17328a3769f3975018257e98d1b23c6caaed07a73c6c35947df4aebdef206b3f
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - jill=jill.__main__:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - wget
+  run:
+    - python >=3.6
+    - fire
+    - jsonschema
+    - python-gnupg
+    - requests
+    - requests-futures
+    - semantic_version
+    - wget
+
+test:
+  imports:
+    - jill
```

```suggestion
test:
  imports:
    - jill
  commands:
    - jill --help  
```
You can still test the command line.